src/ivc/fold_relaxed_plonk_instance_chip.rs

```diff
@@ -779,7 +795,7 @@ mod tests {
 
         let chip = FoldRelaxedPlonkInstanceChip::<C1>::from_instance(
             PlonkInstance {
-                W_commitment: C1::random(&mut rnd),
+                W_commitments: vec![C1::random(&mut rnd)],
```

Comment:
Why only one element?

Reply:
this depends on how you test it. this is just quick hack as I mentioned before.